.golangci.yml

```diff
@@ -128,11 +128,6 @@ issues:
       linters:
         - gomnd
 
-    # https://github.com/go-critic/go-critic/issues/926
-    - linters:
-        - gocritic
-      text: "unnecessaryDefer:"
-
```

Comment:
nit: Would this fit better in a separate PR?

Reply:
I used that to purge the cache :wink: